lib/foreman_theme_satellite/branded_words.rb

```diff
@@ -24,33 +26,14 @@ module ForemanThemeSatellite
     %r{\bhttps://proxy.example.com:8080\b(?!-)} => 'https://proxy.example.com:8080',
     /\bproxy for all outgoing HTTP connections\b(?!-)/ => 'proxy for all outgoing HTTP connections',
     /\bperform various actions through those proxies\b(?!-)/ => 'perform various actions through those proxies',
-    /\bForeman\b(?!-)/       => 'Satellite',
-    /\bforeman\b(?!-)/       => 'satellite',
-    /\bsmart-proxy\b(?!-)/   => 'capsule',
-    /\bSmart-proxy\b(?!-)/   => 'Capsule',
-    /\bsmart proxies\b(?!-)/ => 'capsules',
-    /\bSmart proxies\b(?!-)/ => 'Capsules',
-    /\bsmart-proxies\b(?!-)/ => 'capsules',
-    /\bSmart-proxies\b(?!-)/ => 'Capsules',
-    /\bSmart Proxies\b(?!-)/ => 'Capsules',
-    /\bSmart Proxy\(ies\)(?!-)/ => 'Capsule(s)',
-    /\bSmart Proxy\b(?!-)/   => 'Capsule',
-    /\bsmart proxy\b(?!-)/   => 'capsule',
-    /\bSmart-Proxies\b(?!-)/ => 'Capsules',
-    /\bSmart proxy\b(?!-)/   => 'Capsule',
-    /\bSmart-Proxy\b(?!-)/   => 'Capsule',
-    /\boVirt\b(?!-)/         => 'RHEV',
-    /\bovirt\b(?!-)/         => 'RHEV',
-    /\bOvirt\b(?!-)/         => 'RHEV',
-    /\bFreeIPA\b(?!-)/       => 'Red Hat Identity Management',
-    /\bOpenStack\b(?!-)/     => 'RHEL OpenStack Platform',
-    /\bopenstack\b(?!-)/     => 'RHEL OpenStack Platform',
-    /\bOpenstack\b(?!-)/     => 'RHEL OpenStack Platform',
-    /\bProxy\b(?!-)/         => 'Capsule',
```

Comment:
We still need those, while imprecise string are in the core.

Reply:
I re-added a check for proxy/proxies. OpenStack branding is already covered by line 35.